datalad/distribution/create_sibling.py

```diff
@@ -548,9 +538,12 @@ def create_postupdate_hook(path, ssh, dataset):
         # collate content for post_update hook
         hook_content = '\n'.join(['#!/bin/bash', 'git update-server-info', make_log_dir, json_command])
 
-        with make_tempfile(content=hook_content) as tempf:  # create post_update hook script
-            ssh.copy(tempf, hook_remote_target)             # upload hook to dataset
-        ssh(['chmod', '+x', hook_remote_target])            # and make it executable
+        with make_tempfile(content=hook_content) as tempf:
+            # create post_update hook script
+            # upload hook to dataset
+            ssh.copy(tempf, hook_remote_target)
```

Comment:
I have no idea why (or whether) this worked before, but this is one of the things currently failing (even manually), since the directory to copy to ('hooks') doesn't exist on the remote end.
So, similar to `make_log_dir = ... logs_remote_dir` in line 524 a `make_hooks_dir = ... hooks_remote_dir` would be required, I guess.